ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -221,7 +266,7 @@ func TestCreateEC2Explicitly(t *testing.T) {
 		func(input *ecs.CreateServiceInput) {
 			launchType := input.LaunchType
 			assert.Equal(t, "EC2", aws.StringValue(launchType))
-
+			assert.Nil(t, input.PlatformVersion, "platform version is not nil")
```

Comment:
nice! suggest also adding this same check to one test that is not explicitly setting launchType to EC2 (since it's the default) - maybe `TestCreateWithTaskPlacement`, since task placement is only valid on EC2?